src/main/java/com/softserve/teamproject/service/impl/UserDetailsServiceImpl.java

```diff
@@ -0,0 +1,50 @@
+package com.softserve.teamproject.service.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@Service
+public class UserDetailsServiceImpl implements UserDetailsService {
```

Comment:
If you use javadoc for documentation maybe it is needed to add class comments? This could be discussed with Sergii